benchmark/_interfaces.py

```diff
@@ -36,7 +36,9 @@ def query(filter):
         filter for the fields that are specified in the filter.
 
         :param dict filter: The filter in the JSON compatible format.
-        :return: A Deferred that fires with the results in the JSON format.
+        :param int limit: The number of the *latest* results to return.
```

Comment:
:grey_question: Perhaps I'm just being slow today, but I don't quite follow what this means. Is it "The number of results to return. Results are sorted from most recent to least recent. Only the most recent are returned."?

That seems too wordy, but perhaps something to that extent would be more clear.